com.unity.render-pipelines.high-definition/Samples~/MaterialSamples/Eye.unity

```diff
@@ -123,6 +123,184 @@ NavMeshSettings:
     debug:
       m_Flags: 0
   m_NavMeshData: {fileID: 0}
+--- !u!1 &232594471
+GameObject:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  serializedVersion: 6
+  m_Component:
+  - component: {fileID: 232594474}
+  - component: {fileID: 232594473}
+  - component: {fileID: 232594472}
+  m_Layer: 5
+  m_Name: DescriptionText
+  m_TagString: Untagged
+  m_Icon: {fileID: 0}
+  m_NavMeshLayer: 0
+  m_StaticEditorFlags: 0
+  m_IsActive: 1
+--- !u!114 &232594472
+MonoBehaviour:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_GameObject: {fileID: 232594471}
+  m_Enabled: 1
+  m_EditorHideFlags: 0
+  m_Script: {fileID: 11500000, guid: 9541d86e2fd84c1d9990edf0852d74ab, type: 3}
+  m_Name: 
+  m_EditorClassIdentifier: 
+  m_Material: {fileID: 0}
+  m_Color: {r: 1, g: 1, b: 1, a: 1}
+  m_RaycastTarget: 1
+  m_RaycastPadding: {x: 0, y: 0, z: 0, w: 0}
+  m_Maskable: 1
+  m_OnCullStateChanged:
+    m_PersistentCalls:
+      m_Calls: []
+  m_text: 'The diffusion profiles in this sample are calibrated for one meter eye
+    balls.
+
+    To use them in a real scale scenario, roughly 25mm eye balls, you
+    need to adjust the world scale in the diffusion profiles accordingly.
+
+
+    In
+    addition, Directional lights will have difficulty to light such small objects,
+    instead prefer punctual lights to have correct results.'
```

Comment:
Suggested change: 
Unity calculates the diffusion profiles in this sample for eyeballs that have a diameter of one meter.

In reality, eyeballs have a diameter of roughly 25mm. To use the diffusion profiles at a realistic scale you need to adjust the world scale in the diffusion profiles accordingly. To create lighting that works with such small objects, use punctual lights instead of directional lights.